sdk/ai/Azure.AI.Agents.Persistent/src/Custom/Streaming/CreateRunStreamingOptions.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.AI.Agents.Persistent.Custom.Streaming
+{
+    public class CreateRunStreamingOptions
+    {
+        public CreateRunStreamingOptions() { }
+        /// <summary> The overridden model name that the Agent should use to run the thread. </summary>
+        public string OverrideModelName { get; set; } = default;
+        /// <summary> The overridden system instructions that the agent should use to run the thread. </summary>
```

Comment:
Agent (capital "A"). And elsewhere in ref docs

Reply:
Fixed.